File: java/link-message-client/src/main/java/link/message/client/content/InvokeType.java
package link.message.client.content;

/**
 * 调用类型
 * @author zhongt
 *
 */
public class InvokeType {
	
	/**不调用（仅仅只在消息中心列表显示）*/
	public static final int DO_NOTHING         = 0;
	/**一级调用（直接打开消息里，contents的第一个content中定义的action，目前只支持action_type为app、url和html，如果是none则无反应）*/
	public static final int INVOKE_CUSTOM_APP  = 1;
	/**二级调用（直接打开聊天窗口，展示聊天对话信息）*/
	public static final int INVOKE_CHAT_WINDOW = 2;
}

File: net/Link.Message.Client/content/InvokeType.cs
namespace Link.Message.Client.content
{

	/// <summary>
	/// 调用类型
	/// </summary>
	public enum InvokeType
	{
        /// <summary>
        ///不调用（仅仅只在消息中心列表显示）
        /// </summary>
        DoNothing = 0,

        /// <summary>
        /// 一级调用（直接打开消息里，contents的第一个content中定义的action，目前只支持action_type为app、url和html，如果是none则无反应）
        /// </summary>
		InvokeCustomApp = 1,

        /// <summary>
        /// 二级调用（直接打开聊天窗口，展示聊天对话信息）
        /// </summary>
		InvokeChatWindow = 2
	}
}